.tasks/release.yml

```diff
@@ -0,0 +1,69 @@
+version: '3'
+
+vars:
+  GPG_PRIVATE_KEY: '{{.GPG_PRIVATE_KEY | default ""}}'
+  PASSPHRASE: '{{.PASSPHRASE | default ""}}'
+  GPG_FINGERPRINT: '{{.GPG_FINGERPRINT | default ""}}'
+
+tasks:
+  import-gpg:
+    desc: "Import GPG private key for signing releases"
+    cmds:
+      - |
+        if [ -z "{{.GPG_PRIVATE_KEY}}" ]; then
+          echo "Error: GPG_PRIVATE_KEY environment variable is required"
+          exit 1
+        fi
+        if [ -z "{{.PASSPHRASE}}" ]; then
+          echo "Error: PASSPHRASE environment variable is required"
+          exit 1
+        fi
```

Comment:
This can be replaced with status 

https://taskfile.dev/docs/reference/schema#status